frontend/src/APIClients/EntityAPIClient.ts

```diff
@@ -36,64 +36,46 @@ const create = async ({
     AUTHENTICATED_USER_KEY,
     "accessToken",
   )}`;
-  try {
-    const { data } = await baseAPIClient.post("/entities", formData, {
-      headers: { Authorization: bearerToken },
-    });
-    return data;
-  } catch (error) {
-    return error;
-  }
+  const { data } = await baseAPIClient.post("/entities", formData, {
```

Comment:
We still need to catch the error to ensure it does not propagate up the stack when this is called. In `AuthAPIClient.ts` the functions return null if an error is caught (ex. `login`). 

We could do something like this:
```
const create = async ({
  formData,
}: {
  formData: FormData;
}): Promise<EntityResponse | null> => {
  const bearerToken = `Bearer ${getLocalStorageObjProperty(
    AUTHENTICATED_USER_KEY,
    "accessToken",
  )}`;
  try {
    const { data } = await baseAPIClient.post("/entities", formData, {
      headers: { Authorization: bearerToken },
    });
    return data;
  } catch (e: unknown) {
    return null;
  }
};